website/static/js/registrationUtils.js

```diff
@@ -516,6 +534,15 @@ var RegistrationEditor = function(urls, editorId) {
         }
     });
 
+    self.markAsRequired = function() {
+
+        var questions = self.flatQuestions();
+        $.each(questions, function(idx, question) {
```

Comment:
@samchrisinger this was written mostly for development purposes, if it would not be useful, I will remove it